Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/api/ui/TestForAPIFragment.kt

```diff
@@ -118,19 +118,11 @@ class TestForAPIFragment : Fragment(R.layout.fragment_test_for_a_p_i),
 
     private var lastSetCountries: List<String>? = null
 
+    @SuppressLint("SetTextI18n")
```

Comment:
After your Refactoring this seems to be obsolete.

Reply:
Still needed to stop the warnings regarding hardcoded strings (which are okay because we won't add test texts to the translation process).